src/codeu/chat/client/simplegui/Conversation.java

```diff
@@ -0,0 +1,112 @@
+package codeu.chat.client.simplegui;
+
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Container;
+import java.awt.FlowLayout;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+
+/**
+ * @author  Troy Jones
+ * @date    5/21/17
+ * @brief   Contains all GUI necessary to execute the frame to 
+ *          enable chat.
+ */
+@SuppressWarnings("serial")
+public class Conversation extends JFrame {
+
+
+    // constructor
+    public Conversation() {
+
+        // Used to see if user has clicked inside the editable
+        //  textarea. If true, the default text will clear. If false, the 
```

Comment:
nit: please align with above

Also, it's not clear why you need this in the first place
Does this actually compile? For some reason I thought java only allows you to use final variables inside closures.